cmd/kubectl-gadget/debug.go

```diff
@@ -25,18 +25,16 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 	"sigs.k8s.io/yaml"
-
-	"github.com/inspektor-gadget/inspektor-gadget/cmd/kubectl-gadget/utils"
 )
 
-func getGadgetPodsDebug(client *kubernetes.Clientset) string {
+func getGadgetPodsDebug(client *kubernetes.Clientset, gadgetNamespace string) string {
 	var sb strings.Builder
 
 	listOpts := metav1.ListOptions{
-		LabelSelector: "k8s-app=" + utils.GadgetNamespace,
+		LabelSelector: "k8s-app=gadget",
```

Comment:
I thought it too, but all other places where `k8s-app` is used has `"gadget"` directly hardcoded in it. And I guess it makes sense to leave it at `gadget`.

I think this is okay since this is only a label for the gadget pod itself. And since the DaemonSet (and therefore the pods too) is in its own namespace there is no problem.

This way it is also maybe easier to discover all gadget pods, be it in a single namespace or all namespaces.